src/Bedrock.Framework/Bedrock.Framework.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
This should support both tfms

Reply:
Fixed it.
The only drawback is you can update the `Microsoft.AspNetCore.Http.Connections.Client` package to v5.0.12 and not v6.0.0.